HomeTask_10_dir/HomeTask_10.py

```diff
@@ -0,0 +1,10 @@
+import re
+with open('input.txt', 'r') as inp:
+    data = inp.read()
+    regexp = re.findall(r'(?<![A-Z]{4})(?<=[A-Z]{3})[a-z](?<![A-Z])(?=[A-Z]{3})(?![A-Z]{4})', data)
+    with open('output.txt', 'w') as out:
```

Comment:
There was no task to write smth in files )) This is redundant here. More over if you have values - you can write to file and no further reading required - just use your variable again to print output. This is much faster